src/core/object/controller/controller.ts

```diff
@@ -127,6 +130,13 @@ export default class Controller {
 				type: 'setEditState',
 				fn: (isEditing) => {
 					this.isEditing = isEditing;
+					if (isEditing) {
+						clearTimeout(this.setNotEditingTimeout);
+						this.setNotEditingTimeout = setTimeout(() => {
+							this.isEditing = false;
+							this.eventEmitter.emit('updateEditStatus', false);
+						}, 5000);
+					}
```

Comment:
Should it have a `return`?  Otherwise we will call `this.eventEmitter.emit('updateEditStatus', isEditing);` again? or is that not a problem?